src/sidebar/store/modules/annotations.js

```diff
@@ -388,6 +390,44 @@ function findAnnotationByID(state, id) {
   return findByID(state.annotations, id);
 }
 
+/**
+ * Return the number of page notes.
+ */
+const noteCount = createSelector(
+  state => state.annotations,
+  annotations => {
+    return arrayUtil.countIf(annotations, metadata.isPageNote);
```

Comment:
We don't strictly require this as it requires some judgement about readability, but if an arrow function's body consists of a single `return` statement it can be replaced by the return expression (`annotations => arrayUtil.countIf(annotations, metadata.isPageNote)`)

Reply:
Kinda surprised this wasn't optimized by the formatter.